src/systems/filecoin_mining/storage_mining/_index.md

```diff
@@ -19,36 +19,44 @@ The Filecoin Storage Mining Subsystem ensures a storage miner can effectively co
 The above involves a number of steps to putting on and maintaining online storage, such as:
 
 - Committing new storage (see Sealing and PoRep)
-- Continously proving storage ({see {<sref election_post>}})
+- Continously proving storage
```

Comment:
why the link removal?
```suggestion
- Continously proving storage (see {{<sref election_post>}})
```

Reply:
link was inactive and it has been restored.